ansible/roles/nemea/tasks/install.yml

```diff
@@ -7,6 +7,15 @@
 - name: Install NEMEA packages
   yum: "name={{ packages }} state=installed"
 
+- name: Create NEMEA reports path
+  file:
+    path: "{{ nemea_reporters.path }}"
```

Comment:
I do not see the nemea_reporters.path in my configuration. If it is a user-defined variable, it should be somehow reflected in documentation and the value of nemea_reporters_config_path, which is used elsewhere.